packages/office-ui-fabric-react/src/components/Button/DefaultButton/DefaultButton.styles.ts

```diff
@@ -12,11 +12,17 @@ import {
   getStyles as getSplitButtonStyles
 } from '../SplitButton/SplitButton.styles';
 
+import {
+  primaryStyles,
+  standardStyles
+} from '../ButtonThemes';
+
 const DEFAULT_BUTTON_HEIGHT = '32px';
 const DEFAULT_BUTTON_MINWIDTH = '80px';
 const DEFAULT_PADDING = '0 16px';
 
 export const getStyles = memoizeFunction((
+  primary: boolean,
```

Comment:
I think we should be consistent here and pass in theme first, then custom styles if applicable, then states if applicable.